test/bin/ci_phase_iso_build.sh

```diff
@@ -153,7 +166,13 @@ else
     if ! ${GOT_CACHED_DATA} ; then
         echo "WARNING: Build cache is not available, rebuilding all the artifacts"
     fi
-    run_image_build
+
+    # Optionally run bootc image builds
+    if ${COMPOSER_CLI_BUILDS} ; then
+        run_image_build
+    else
+        run_bootc_image_build
```

Comment:
Is this short-term plan until we port all images? Seems we'll be spinning metal-host for single test?

Reply:
In the initial implementation stage, I thought, it would be best to separate bootc and composer-cli flows until we stabilize bootc. Then, we can go back and revisit how or if at all we should combine those.